terraform/test/main.tf

```diff
@@ -233,7 +233,8 @@ module "cloudfront_distributions" {
       aliases             = ["cpm.${var.primary_domain}"]
       acm_certificate_arn = module.communities_only_ssl_certs.cloudfront_certs["cpm"].arn
     }
-    geo_restriction_countries = ["GB", "IE"]
+    # So GitHub Actions can access for end to end tests
+    geo_restriction_countries = null
```

Comment:
Doesn't really matter since it's just the test env, but couldn't you have done `["GB", "IE", "US"]`?

Reply:
Possibly, do GitHub actions guarantee they'll run in the US?